src/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -233,10 +238,202 @@
     <Compile Include="$(CommonPath)\Interop\Windows\user32\Interop.Win32SystemColors.cs">
       <Link>Common\Interop\Windows\user32\Interop.Win32SystemColors.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)/System/Drawing/ColorConverterCommon.cs">
+    <Compile Include="$(CommonPath)\System\Drawing\ColorConverterCommon.cs">
       <Link>System\Drawing\ColorConverterCommon.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)/System/Drawing/ColorTable.cs">
+    <Compile Include="$(CommonPath)\System\Drawing\ColorTable.cs">
+      <Link>System\Drawing\ColorTable.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\System\Drawing\KnownColorTable.cs">
+      <Link>System\Drawing\KnownColorTable.cs</Link>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetsUnix)' == 'true'">
+    <Compile Include="mono\System.Drawing\Bitmap.cs" />
```

Comment:
Could we use a directory name other than "mono", e.g. "Unix" or something like that to match what's used elsewhere in corefx?